packages/dd-trace/src/llmobs/tagger.js

```diff
@@ -89,10 +91,15 @@ class LLMObsTagger {
 
     this._setTag(span, ML_APP, spanMlApp)
 
-    const parentId =
-      parent?.context().toSpanId() ??
-      span.context()._trace.tags[PROPAGATED_PARENT_ID_KEY] ??
-      ROOT_PARENT_ID
+    let parentId
+    if (parent instanceof Identifier) {
```

Comment:
Why is an identifier as the parent needed?

Reply:
ah actually i had it this way because i don't think i have a reference to the parent span at the point of the vercel ai integration for llmobs parentage, just the current span, and on it, its parent id, which i opted to pass in here (but, correct me if i'm wrong about spans holding a reference to their entire parent span)